web/packages/teleport/src/Sessions/SessionList/SessionJoinBtn.tsx

```diff
@@ -84,6 +86,17 @@ export const SessionJoinBtn = ({
         showCTA={showCTA}
         closeMenu={closeMenu}
       />
+      {showModeratedCTA && (
+        <Box mx="12px" my="3">
```

Comment:
```suggestion
        <Box mx={12} my={3}>
```

Reply:
I don't think that setting `mx={12}` is a good idea. It works only because there is no value with index 12 in our spacing array, so it falls back to pixels.
It's confusing to use this way, because:
`mx={11}` -> margin 80px
`mx={12}` -> margin 12px

Let's go back to `12px`.

Reply:
Ah you're right. Makes sense. Good catch!